docs/common-bugs.rst

```diff
@@ -0,0 +1,278 @@
+:Solution: This error occurs when ``initrd.img``, ``vmlinuz`` and the repository (or stage2) are
+    not from the same media or location.
+:Example: `rhbz#1169034 <https://bugzilla.redhat.com/show_bug.cgi?id=1169034>`_
+
+Changes in Live OS
+^^^^^^^^^^^^^^^^^^
+
+:Detection: The Live OS requires changes.
+:Solution: Reassigning to spin-kickstarts.
+
+Changes in boot.iso
+^^^^^^^^^^^^^^^^^^^
+
+:Detection: The ``boot.iso`` requires changes.
+:Solution: Reassigning to lorax.
+
+Icon issues
+^^^^^^^^^^^
+
+:Detection: The Anaconda icons in Live OS requires changes.
+:Solution: Reassigning to fedora-logos
+:Example: `rhbz#1699034 <https://bugzilla.redhat.com/show_bug.cgi?id=1699034>`_
+
+Font issues
+^^^^^^^^^^^
+
+:Detection: In the Welcome spoke, there are replacement glyphs (rectangles) instead of
+    characters in a name of a language. This usually means that that there is no font for this
+    language installed in the installation environment.
+:Solution: Reassigning to lorax or spin-kickstarts.
+:Example: `rhbz#1530086 <https://bugzilla.redhat.com/show_bug.cgi?id=1530086>`_
+
+Payload issues
+--------------
+
+These issues are related to the content that is installed on the target system.
+
+Non-fatal POSTIN scriptlet failure
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+:Detection: The package installation fails with a message ``Non-fatal POSTIN scriptlet failure in
+    rpm package``. The failing package has to fix its scriptlet, because `all scriptlets
+    MUST exit with the zero exit status.
+    <https://docs.fedoraproject.org/en-US/packaging-guidelines/Scriptlets/>`_
+:Solution: All RPM errors are fatal during the installation (see the bug 1565123). Reassigning.
+:Example: `rhbz#1588409 <https://bugzilla.redhat.com/show_bug.cgi?id=1588409>`_
+
+Changes in package groups and environments
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+:Detection: The reporter wants a new package to be installed by default.
+:Solution: Reassigning to comps.
+:Example: `rhbz#1787018 <https://bugzilla.redhat.com/show_bug.cgi?id=1787018>`_
+
+Corrupted ISO
+^^^^^^^^^^^^^
+
+:Detection: The package installation fails with a message ``Some packages from local repository
+    have incorrect checksum``. This happens when the packages cannot be accessed, because they
+    are located on a corrupted ISO or an unmounted device.
+:Solution: The ISO might be corrupted. Please, try to download it again and verify the checksum.
+:Example: `rhbz#1551311 <https://bugzilla.redhat.com/show_bug.cgi?id=1551311>`_
+
+Issues with live payload
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+:Detection: The image installed by the live OS payload requires changes.
+:Solution: Anaconda doesn't create the live image. Reassigning to spin-kickstarts.
+
+Issues with OSTree
+^^^^^^^^^^^^^^^^^^
+
+:Detection: The installation with the OSTree payload fails.
+:Solution: It might be related to the OSTree payload. Reassigning to Colin Walters.
+:Contact: `walters@redhat.com <mailto:walters@redhat.com>`_
```

Comment:
I would remove this. First it's code in our code-base and the problem could be there. Second, we don't want to serve Colins mail for spam bots.

Reply:
Fixed.